Readme.md

```diff
@@ -0,0 +1,171 @@
+The environment variables are passed to the caller workflows as either inputs or secrets. 
+Whenever you want to add a new environment variable to your repository and pass it as a secret, you should update your workflow as well. 
+
+For this purpose, you need to perform the following steps. 
+#### 1. Update inputs and secrets in a reusable workflow definition
+```
+on:
+  workflow_call:
+    inputs:
+      input-variable:
+        required: true
+        type: string
+    secrets:
+      secret-variable:
+        required: true
+```
+
+#### 2. Reference the input or secret that you defined in the on key in the previous step.
+
+```
+jobs:
+  reusable_workflow_job:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ./.github/workflows/...
+        with:
+          input-variable: ${{ inputs.input-variable }}
+          secret-variable: ${{ secrets.secret-variable }}
+```
+
+#### 3. Include the secret in the caller workflow. 
+- Use the `with` keyword to pass named inputs.
+- Use the `secrets` keyword to pass named secrets. 
+````
+jobs:
+  call-workflow-passing-data:
+    uses: octo-org/example-repo/.github/workflows/reusable-workflow.yml@main
+    with:
+      input-variable: string
+    secrets:
+      secret-variable: ${{ secrets.SECRET_VARIABLE }}
+````
+
+#### 4. Create secrets
+Create a new secret for the specific repository, with the same name you specified in the previous step.
+1. Navigate to the main page of the repository. 
+2. Under the repository name, click Settings
+
+    ![Settings tab](assets/images/repo-actions-settings.png)
+
+3. In the left sidebar, click  Secrets > Actions.
+
+    ![Secrets ](assets/images/secrets.png)
+4. Click New repository secret.
+5. Type a name for your secret in the Name input box.
+
+    ![New secret](assets/images/settings-new-secret.png)
+6. Enter the value for your secret.
+7. Click Add secret.
+
+## Why use Github Actions
+The use of Github Actions simplifies the automation process. Actions are similar to a plugin that comes bundled for every Github repository created, and that can execute any desired task.
+
+Github Actions embrace five underlying concepts: jobs, workflows, events, actions and runners. 
+
+
+## Why use workflows
+
+As stated in the [Github Docs](https://docs.github.com/en/actions/using-workflows/about-workflows):
+
+>A workflow is a configurable automated process that will run one or more jobs. Workflows are defined by a YAML file checked in to your repository and will run when triggered by an event in your repository, or they can be triggered manually, or at a defined schedule.
+
+## Why use reusable workflows
+
+As stated in the [Github Docs](https://docs.github.com/en/actions/using-workflows/reusing-workflows): 
+>The use of reusable workflows avoids duplication and makes workflows easier to maintain. This way workflows can be created more quickly. 
+
+The main goal for Graasp Deploy is to build up a library of reusable workflows that can be centrally maintained.
+
+
```

Comment:
I'd like to see a section for "Why refer to a fixed workflow commit" since I might forget^^

Reply:
Done